Update by-covid.mdx (#94)
ELIXIR-NO/website: src/content/projects/by-covid.mdx

---
title: "BY-COVID"
summary: "BY-COVID aims to make data about SARS-CoV-2 and other viruses easier to access, aggregate and analyse"
status: "completed"
category: "European project"
funder:
    name: "European Commission"
    link: https://ror.org/00k4n6c32
project_number:
    grant_agreement_id: 101046203
    link: https://doi.org/10.3030/101046203
period: 2021-10 2024-09
external_link: https://by-covid.org/

> keywords: ["pathology", "covid", "brokering"]

Reviewer: ```suggestion
keywords: ["pathogen", "covid", "brokering"]
```

Reviewer: Changing keyword based on Erik's contribution [here](https://raw.githubusercontent.com/ELIXIR-NO/website/refs/heads/main/src/content/projects/PDN.mdx)

---
BY-COVID is a project funded under the European Union’s Horizon Europe Research and Innovation Programme. 

> It aims to make data about SARS-CoV-2 and other viruses easier to access, aggregate and analyse. 

Reviewer: ```suggestion
It aims to make data about SARS-CoV-2 and other viruses easier to access, aggregate, and analyse. 
```

This will enable scientists to respond faster to new strains of SARS-CoV-2 or to new viruses. 
It will also help policy-makers assess the impact of the disease and take the most appropriate measures to protect people against new viral diseases.

> ELIXIR Norway's contribution has been to contribute to work package 1 on support for virological analyses in emerging disease outbreaks (UiT). The Centre for Information Technology (USIT) at the University of Oslo has been contributing to work package 5 on facilitating secondary use of COVID-19 vaccine trial data and biosamples to test existing vaccines against emerging variants, using TSD as the data repository to access trial data.

Reviewer: ```suggestion
ELIXIR Norway contributes to work package 1 on support for virological analyses in emerging disease outbreaks (UiT). The Centre for Information Technology (USIT) at the University of Oslo contributes to work package 5 on facilitating secondary use of COVID-19 vaccine trial data and biosamples to test existing vaccines against emerging variants, using TSD as the data repository to access trial data.
```

Reviewer: As agreed at the last meeting, we should try to keep the description in the present tense.
